src/libos/src/util/mod.rs

```diff
@@ -5,5 +5,6 @@ pub mod log;
 pub mod mem_util;
 pub mod mpx_util;
 pub mod random;
+pub mod resolv_conf_util;
```

Comment:
Although I wanted to point out that encapsulating all resolve_conf related stuff into a separate module is a good idea, I kept it to myself as I thought the original implementation was acceptable. Now that you have added the `resolv_conf_util` module,  I believe you also see the benefit of reconstructing all resolve_conf related code as a module.  **If a resolve_conf module is to be added, then it should be done properly: give it well-designed public APIs and hide all implementation details.** A good example is `crate::util::log`, which is initialized at the startup of the enclave (as the resolve_conf module would do) and used later by other modules (as the resolve_conf module would do, too).

Reply:
Done